packages/react-core/src/components/TextInputGroup/__tests__/TextInputGroup.test.tsx

```diff
@@ -1,166 +1,87 @@
 import React from 'react';
-import { render } from '@testing-library/react';
-import { mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
 import { TextInputGroup, TextInputGroupContext } from '../TextInputGroup';
-import { TextInputGroupMain } from '../TextInputGroupMain';
-import { TextInputGroupUtilities } from '../TextInputGroupUtilities';
-import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
 
 describe('TextInputGroup', () => {
-  test('renders content', () => {
-    const view = render(
-      <TextInputGroup>
-        <TextInputGroupMain>Foo</TextInputGroupMain>
-        <TextInputGroupUtilities />
-      </TextInputGroup>
-    );
-    expect(view.container).toMatchSnapshot();
-  });
-
-  test('gets custom class and id', () => {
-    const view = render(
-      <TextInputGroup className="custom-class" id="test-id">
-        <TextInputGroupMain>Foo</TextInputGroupMain>
-        <TextInputGroupUtilities />
-      </TextInputGroup>
-    );
-    expect(view.container).toMatchSnapshot();
-  });
+  it('renders children', () => {
+    render(<TextInputGroup>Test</TextInputGroup>);
```

Comment:
Good point! What kind of assertion do you think would be appropriate for checking rendering without children though?

Part of me leans towards asserting that `Test` isn't visible so that we know it doesn't otherwise appear in the component in the test for children, but that doesn't really test that it renders as we expect without children. The other part of me thinks that maybe this is where the snapshot test for the component should be used.

Reply:
In this case I might resort to a `data-testid` which gets applied to the parent `div`. If we `queryByTestId` and verify it is null, then that should cover us verifying the entirety of the component is not rendered.

A snapshot could be used but would be less specific IMO.

Reply:
I'm a bit confused, shouldn't querying by the testid select the empty div with the associated TextInputGroup styling?

I could add a testid as you suggest and verify that it renders that div without children though, which I think would accomplish the overall goal here if I've understood you right.

Reply:
Yeah that's what I meant, sorry haha. I forgot when looking at your comment this morning that it was meant to render without children. What I said is if the component was meant to not be shown without children.